docs/hub/datasets-viewer.md

```diff
@@ -37,6 +37,21 @@ In this case, an informational message lets you know that the Viewer is partial.
 
 To power the dataset viewer, the first 5GB of every dataset are auto-converted to the Parquet format (unless it was already a Parquet dataset). In the dataset viewer (for example, see [`datasets/glue`](https://huggingface.co/datasets/glue)), you can click on [_"Auto-converted to Parquet"_](https://huggingface.co/datasets/glue/tree/refs%2Fconvert%2Fparquet/cola) to access the Parquet files. Please, refer to the [Datasets Server docs](/docs/datasets-server/parquet_process) to learn how to query the dataset parquet files with libraries such as Polars, Pandas or DuckDB.
 
+<Tip>
```

Comment:
Nice! I think if we want to use it as a `<Tip>` here, I would keep it shorter to maybe just 2-3 sentences and let the linked content do the heavy lifting of explaining in more detail the advantages :)

Reply:
done, thanks